server/src/metadata.rs

```diff
@@ -155,6 +180,24 @@ pub mod entities_metadata_query {
                     .and_then(|title| title.as_str())
                     .map(|title| title.to_string());
                 let id = get_iri(result.id as i32);
+
+
+                let authors_map: HashMap<String, String> = serde_json::from_value(result.authors.unwrap_or_default())
+                        .unwrap_or_default();
+
+                let creators: Vec<Creator> = authors_map.iter()
+                    .map(|(id, username)| Creator {
+                        creator_type: CreatorType::Person,
+                        // Id is a url that links to our authors. It looks like
+                        // the following
+                        // https://serlo.org/user/:userId/:username e.g
+                        // https://serlo.org/user/240298/felix_eccardt
```

Comment:
```suggestion
```

Reply:
> I changed the comment but left it in as for me id is badly chosen key by amb, it's not clear to me that a URL is behind this.

By the way: This does not come from AMB but from JSON-LD -> see the `@id` keyword there.